dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -94,6 +94,22 @@ When a set of routes is created in various projects, the overall set of routes
 is available to the set of routers. Each router admits (or selects) routes from
 the set of routes. By default, all routers admit all routes.
 
+As of {product-title} 3.6, routers can accept IPV6 traffic from outside the
+cluster for the following routes:
+
+* Route type 1
```

Comment:
@imcsk8 Can you specify what route types this works for?  Is it for all of them?  I.e. passthrough too?  I would expect so, but please clarify.